src/types/domain/Player.ts

```diff
@@ -2,7 +2,17 @@ import { Match } from "./Match";
 
 export type Player = {
     name: string;
+    /**
+     * A collection of Match objects that the player has participated in. This includes all matches regardless of their validity.
+     */
     matches: Match[];
+    /**
+     * Number of games we want to take into account when calculating stats (i.e. 4 player pods).
```

Comment:
nit: you can just copy the comment I suggested from the Commander.ts here as well